internal/analytics/segment_reporter.go

```diff
@@ -59,10 +62,15 @@ func (r *SegmentReporter) RegisterCurrentIdentity(ctx context.Context, k8sCli ku
 
 // ReportCommand reports a new executed command. The command should be anonymized before using this method.
 // The RegisterCurrentIdentity needs to be called first.
-func (r *SegmentReporter) ReportCommand(platform config.CommPlatformIntegration, command string) error {
+func (r *SegmentReporter) ReportCommand(platform config.CommPlatformIntegration, command string, isInteractiveOrigin bool) error {
+	origin := typedCommand
+	if isInteractiveOrigin {
+		origin = buttonCommand
+	}
 	return r.reportEvent("Command executed", map[string]interface{}{
 		"platform": platform,
 		"command":  command,
+		"origin":   origin,
```

Comment:
WDYT about `trigger`?

```suggestion
		"trigger":   trigger,
```

Reply:
Honestly, I like the origin even more 🤔 I discuss that also with Blair before implementing and he also was OK with this property name.